Modules/Registration/Common/include/itkLandmarkBasedTransformInitializer.h

```diff
@@ -76,7 +76,7 @@ namespace itk
  * and is described in the Insight Journal Article
  * "Affine Transformation for Landmark Based Registration Initializer
  * in ITK" by Kim E.Y., Johnson H., Williams N.
- * available at  http://midasjournal.com/browse/publication/825
+ * available at  https://midasjournal.com/browse/publication/825
```

Comment:
Neither one works here for me, at the moment (here in the Netherlands). A slightly different URL, https://www.midasjournal.org/browse/publication/825 does work for me.